packages/calcite-components/src/components/tip/tip.stories.ts

```diff
@@ -51,12 +52,12 @@ const createAttributes: (options?: { exceptions: string[] }) => Attributes = ({
   );
 };
 
-const html = `<img slot="thumbnail" src="${placeholderImage({
+const htmlSnippet = `<img slot="thumbnail" src="${placeholderImage({
```

Comment:
Nitpick: can this be renamed to better express its contents? It looks like it's simple tip content?